src/parser/parse_tree.h

```diff
@@ -3870,8 +3870,7 @@ enum cdc_ddl_type
   CDC_ALTER,
   CDC_DROP,
   CDC_RENAME,
-  CDC_TRUNCATE,
-  CDC_TRUNCATE_CASCADE
```

Comment:
Why is this removed?

Reply:
It is unrelated change. 
I removed it because `TRUNCATE_CASCADE` is obsolete.
I will revert it, because it will be treated In other issue that add some more ddl types.